testsuite/features/secondary/min_ssh_tunnel.feature

```diff
@@ -94,7 +89,7 @@ Feature: Register a Salt system to be managed via SSH tunnel
     Then "ssh_minion" should not be registered
 
   Scenario: Cleanup: register a Salt minion after SSH tunnel tests
-    When I bootstrap "ssh_minion" using bootstrap script with activation key "1-SUSE-SSH-KEY-x86_64" from the proxy
+    When I bootstrap "ssh_minion" using bootstrap script with activation key "1-SUSE-SSH-KEY-x86_64" from the server
```

Comment:
Why do we use the server instead of the proxy here?

Reply:
This is a cleanup step, so we don't really care if we use the server or the proxy, but to answer your question, this is changed to `# workaround for bsc#1220864
`

Basically, mgr-bootstrap command is not available in the proxy container anymore, only  in the server container. They want to figure out a better implementation, returning the script via API instead of generating it through the command mgr-bootstrap.